client/my-sites/stats/stats-post-anniversaries/index.jsx

```diff
@@ -0,0 +1,209 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { localize, moment } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import ErrorPanel from '../stats-error';
+import StatsList from '../stats-list';
+import StatsListLegend from '../stats-list/legend';
+import SectionHeader from 'components/section-header';
+import StatsContentText from '../stats-module/content-text';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+import Card from 'components/card';
+import QueryPosts from 'components/data/query-posts';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSitePostsForQuery, isRequestingSitePostsForQuery } from 'state/posts/selectors';
+
+// Maximum number of simultaneous queries to fetch the posts by year
+const MAX_POST_QUERIES = 10;
+
+const StatModulePostAnniversaries = props => {
+	const { oldestPostQuery, postsByYearQueries, postsByYear, requesting, siteId, translate } = props;
+
+	const allPosts = Array.prototype.concat( ...postsByYear );
+
+	const cardClasses = classNames( 'stats-module', 'is-expanded', 'summary', {
+		'is-loading': requesting,
+		'has-no-data': ! allPosts.length,
+		'is-showing-error': ! allPosts.length,
+	} );
+
+	return (
+		<div>
+			<SectionHeader
+				label={ translate( 'Anniversaries', {
+					comment: 'Title of the anniversaries module',
+				} ) }
+			/>
+			<Card className={ cardClasses }>
+				{ siteId && <QueryPosts siteId={ siteId } query={ oldestPostQuery } /> }
+				{ siteId &&
+					postsByYearQueries.map(
+						( query, i ) => <QueryPosts key={ i } siteId={ siteId } query={ query } />,
+					) }
+
+				<StatsModulePlaceholder isLoading={ requesting } />
+
+				{ ! requesting &&
+					allPosts &&
+					( allPosts.length === 0
+						? <EmptyMessage />
+						: <PostsList allPosts={ allPosts } postsByYear={ postsByYear } /> ) }
+			</Card>
+		</div>
+	);
+};
+
+const CalendarIcon = () => (
+	<span style={ { position: 'relative', top: '-2px' } }>
```

Comment:
Please don't do this, and don't inline style. Style should be in its separate SCSS file.

If it's a more general issue, just open a separate issue.